core/modules/image/image.module

```diff
@@ -881,16 +881,18 @@ function image_style_transform_dimensions($style_name, array &$dimensions) {
  *   An image style array.
  */
 function image_style_flush($style) {
-  // Delete the style directory in each registered wrapper.
-  $wrappers = file_get_stream_wrappers(STREAM_WRAPPERS_WRITE_VISIBLE);
-  foreach ($wrappers as $wrapper => $wrapper_data) {
-    if (file_exists($directory = $wrapper . '://styles/' . $style['name'])) {
-      file_unmanaged_delete_recursive($directory);
+  if (!empty($style)) {
```

Comment:
Hm, this seems like it _should_ through a PHP warning. `$style` should always be something. If it's being called with `NULL` somewhere, we should fix that call, not suppress the warning.